packages/server/lib/util/cookies.ts

```diff
@@ -16,8 +35,22 @@ interface CookieData {
 export class CookieJar {
   _cookieJar: ToughCookieJar
 
-  static parse (cookie) {
-    return Cookie.parse(cookie)
+  static parse (cookie: string) {
+    const toughCookie = Cookie.parse(cookie)
+
+    if (!toughCookie) return
+
+    // fixes tough-cookie defaulting undefined/invalid SameSite to 'none'
+    // https://github.com/salesforce/tough-cookie/issues/191
+    const hasUnspecifiedSameSite = toughCookie.sameSite === 'none' && !sameSiteNoneRe.test(cookie)
```

Comment:
Should we consider contributing this fix to toughCookie? Is this something other people would want too?